mysql-test/suite/binlog_encryption/encrypted_master_switch_to_unencrypted_coords.test

```diff
@@ -136,6 +136,8 @@ start slave;
 SHOW TABLES;
 
 --disable_connect_log
+# IO thread is stopped, stop SQL thread only
+--let $rpl_only_running_threads= 1
```

Comment:
I think using `rpl_only_running_threads=1` is fine to keep as is (in the context of your current commit), but for future reference, there is `--source include/stop_slave_sql.inc` and `stop_slave_io.inc` which would set the behavior expectation more explicitly.